### DISCUSSION TITLE
Issue #4005

### DISCUSSION THREAD

**Discussion by AHTei** [microsoft-search/pnp-modern-search#4012] [Q&A] [answered]
We have raised an issue and provided requested information, but we do not get timely replies.
Please provide assistance on the described issues.
https://github.com/microsoft-search/pnp-modern-search/issues/4005

**Comment by kasperbolarsen** [comment]
Hello, it seems you have misunderstood something. This is not the Microsoft premium support, and there is NO SLA whatsoever.
We are all volunteer working on this project in our sparetime, and the level of support you are entitled to is proportional to the amount of money you paid for this...... none

**Reply by AHTei** [reply]
Hi Kasper,
I understand there is no SLA and it is based on volunteer support offered by the contributors of the product. 
Furthermore, I was not aware the level of support we get would be proportional to the amount of money paid or how the payment is managed. 
Is it membership/subscription based or onetime lump sum payment? I appreciate if proper references could be share. 
Anyways, how should we set expectations for get response to the raised issues. Is it in days, weeks, or months?
Thanks

**Reply by kasperbolarsen** [reply, marked as answer]
Hi again. The level of support you are entitled to is proportional to the price you pay, and since it is free, there is no reason to expect a "timely reply". We will reply once we have some free time available. This might be on the same day as the question is asked, and it might be 4 weeks or 6 months later.
If the question is time sensitive, I would recommend contacting one of the consulting firms that writes about PnP Modern Search, and see if they are able to assist you.
